### PR TITLE
e2e: with the newest version of Selenium, manually updating Geckodriver and Chromedriver is no longer necessary

### PR DESCRIPTION
**Pull request by HowardBraham**
## **Description**

Yay!  We don't have to manually update Geckodriver and Chromedriver anymore, so they shouldn't get out of sync!
https://www.selenium.dev/blog/2023/whats-new-in-selenium-manager-with-selenium-4.11.0/

## **Manual testing steps**

`yarn`

then
```
yarn test:e2e:single <pick_a_testfile> --browser=firefox --debug
yarn test:e2e:single <pick_a_testfile> --browser=chrome --debug
```

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [x] What problem this PR is solving.
  - [x] How this problem was solved.
  - [x] How reviewers can test my changes.
- [ ] I’ve indicated what issue this PR is linked to: Fixes #???
- [x] I’ve included tests if applicable.
- [x] I’ve documented any added code.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by PeterYinusa — DISMISSED** [review]

**Review by DDDDDanica — DISMISSED** [review]

**Review by brad-decker — CHANGES_REQUESTED** [review]
Curious if this has any security implications assuming selenium is downloading a version of chrome that coincides with our driver version?

File: package.json
Comment (HowardBraham) [reply package.json:19]: I was having trouble running local tests on this build without this, so I made it match the other `start` commands.  But do you think there's a problem?

Comment (brad-decker) [reply package.json:19]: usually those issues revolve around lavamoat policy not being updated, etc... I'm hesitant on removing here without confirmation from @PeterYinusa who i think uses this to get as close to our test build as possible which does include lavamoat.

**Comment by HowardBraham** [comment]
Trying @metamaskbot update-policies

**Comment by HowardBraham** [comment]
@metamaskbot update-policies

**Comment by HowardBraham** [comment]
> Curious if this has any security implications assuming selenium is downloading a version of chrome that coincides with our driver version?

With this PR, Selenium is now downloading Chrome and Chromedriver, or Firefox and Geckodriver, on whatever version is considered "stable."

File: package.json
Comment (PeterYinusa) [reply package.json:19]: IMO it's OK for the `start:test` script to be similar to `start` as it is only used for local test development and execution. We're using `build:test` in the pipeline so it won't effect the runs there.

**Review by pedronfigueiredo — DISMISSED** [review]

**Comment by brad-decker** [comment]
@MetaMask/supply-chain could I get a review of security implications with having an automated download in our CI chain?

**Comment by HowardBraham** [comment]
Had to solve the merge conflict after #21543, and it reset approvals

**Comment by brad-decker** [comment]
A member of the team vouched for the maintainer of the tool that installs so i vote we move forward.

**Review by brad-decker — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**
